app/views/request/_sidebar.html.erb

```diff
@@ -2,17 +2,6 @@
   <div class="new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
-  <div id="track-request" class="track-request">
-    <h2><%= _('Follow this request') %></h2>
-
-    <p>
-      <%= n_("There is {{count}} person following this request",
-             "There are {{count}} people following this request",
-             @follower_count,
-             :count => @follower_count) %>
-    </p>
```

Comment:
We are yeah. As we discussed in the original plan (https://github.com/mysociety/alaveteli/issues/3336#issuecomment-228323347) 'follow' isn't a high priority for us on this page.

We could look into a small label maybe in the action menu that shows how many are following?